minikube-gradle-plugin/src/main/java/com/google/cloud/tools/minikube/gradle/MinikubeTask.java

```diff
@@ -0,0 +1,111 @@
+package com.google.cloud.tools.minikube.gradle;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import org.gradle.api.DefaultTask;
+import org.gradle.api.GradleException;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.TaskAction;
+
+/** Generic Minikube task. */
+public class MinikubeTask extends DefaultTask {
+
+  /**
+   * The minikube executable, can include the full path to minikube, default assumes it is on path.
+   * I'm not sure this works on windows, ask someone about this.
+   */
```

Comment:
I think we install as `minikube.exe` on windows.  Not sure if this complicates things a bit on your end. @aaron-prindle knows more.

Reply:
@aaron-pindle @r2d4 any idea here?

I think we might need to launch using "cmd \c minikube.exe"

Reply:
I think it needs to be launched in powershell?  I'm actually not too sure on this one.  Feel free to leave it out for now if its tricky.

Reply:
The binary is `minikube.exe` on Windows.  If users try to run minikube using HyperV then it does need to be run through Powershell but for Virtualbox, any shell will work.

Reply:
I think we should just default to powershell on windows then, so that it supports both hypervisors.